assets/javascripts/discourse/templates/components/babble-composer.hbs

```diff
@@ -1,4 +1,4 @@
 <div class="babble-composer">
-  {{textarea value=text}}
-  <button {{action 'submit'}} {{bind-attr disabled="submitDisabled"}} class="btn btn-primary pull-right">Send</button>
+  {{textarea value=text placeholder=i18nPlaceholder }}
```

Comment:
I think you can just do the following here, without using a component function?
```
title="{{i18n 'babble.placeholder'}}"
```

Like here:
https://github.com/discourse/discourse/blob/master/app/assets/javascripts/admin/templates/customize_colors.hbs#L63